.gitignore

```diff
@@ -1,3 +1,5 @@
 /target
 Cargo.lock
 .vscode/
+sludge-fmod-sys/target
+sludge-fmod/target
```

Comment:
these shouldn't exist as long as things are in a workspace... but i guess it's a good idea.